validator/webui/index.html

```diff
@@ -456,6 +456,11 @@
           button.addEventListener('click', function() {
             var urlToFetch = input.value.trim();
             if (urlToFetch.length > 0) {
+              if (!(urlToFetch.indexOf('http://') == 0 ||
+                    urlToFetch.indexOf('https://') == 0)) {
+                urlToFetch = 'https://' + urlToFetch;
```

Comment:
Per our discussion, lets move forward with this and address the other items in a future PR.

LGTM